### DISCUSSION TITLE
unable to access codespace

### DISCUSSION THREAD

**Discussion by sujeetsahoo** [community/community#84911] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![Screenshot 2024-01-08 115344](https://github.com/community/community/assets/154114647/4f2d36e5-66b5-402e-b3dc-2e3359b5fbcc)
Hello ,
i can't access my codespae,since 2 day.codespae setup is run but it is not proper execute.
it will shows blank setup codespace page.
Best regards.

**Comment by ibra-kdbra** [comment]
1. Browser Cannot Connect: Sometimes you may not be able to access a code space from your browser. If this happens, go to [github.com/codespaces](https://github.com/codespaces) and try connecting to the code space from that page.
 
2. Check Your Internet Connection
 
3. Check Service Availability: If you still can’t connect and the message you’re seeing isn’t in this list, check the service availability of Codespaces at [githubstatus.com](https://www.githubstatus.com/),
 see this [doc](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces)
 
4. Try a Different Browser or Device.

**Reply by sujeetsahoo** [reply]
thanks now it's working.

**Reply by ibra-kdbra** [reply]
Good for you, please mark this question as completed, to let others know the issue has been solved.

**Reply by sujeetsahoo** [reply]
Hi,
same issue occur, can you solve this problem

**Reply by ibra-kdbra** [reply]
Did you try any of the past steps?
If the issue persists, I think it will be a bug from code-space to GitHub

**Reply by sujeetsahoo** [reply]
yes i tred so many times

**Reply by ibra-kdbra** [reply]
I don't have any solution, I guess it's GitHub, search for it online, see if someone is having the issue and confirm if the GitHub is the problem or not.
